src/Rockets.test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
- [ ] Great job working on this capstone. Kindly take this file _(Rockets.test.js)_ to the `_test_` folder

Reply:
done